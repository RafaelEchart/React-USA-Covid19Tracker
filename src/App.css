@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap');

@font-face {
    font-family: "gillsans";
    src:
      local("gillsans"),
      url("./assets/gillsans.ttf") format("truetype");
  }
  
  @font-face {
    font-family: "gillsans-bold";
    src:
      local("gillsans-bold"),
      url("./assets/gillsansbold.otf") format("truetype");
  }
  