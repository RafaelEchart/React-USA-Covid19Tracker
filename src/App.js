import React from 'react';
import {
  Routes,
  Route,
} from 'react-router-dom';
import './App.css';
import 'antd/dist/antd.css';
import SelectedCountryPage from './components/SelectedCountryPage/SelectedCountryPage';
import HomePage from './components/HomePage/HomePage';

function App() {
  return (
    <div className="App">
        <div className='contentContainer'>
          <Routes>
            <Route path="/" element={<HomePage />} />
            <Route path="/:idState" element={<SelectedCountryPage />} />
            {/* <Route path="/myProfile" element={<MyProfile />} /> */}
          </Routes>
        </div>
    </div>
  );
}

export default App;
