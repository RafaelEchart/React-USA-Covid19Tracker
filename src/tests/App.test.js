import React from 'react';
import '@testing-library/jest-dom';
import { BrowserRouter } from 'react-router-dom';
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import store from '../redux/configureStore';
import App from '../App';

describe('App component renders as intended', () => {
  it('App component matches the snapshot', () => {
    const tree = renderer.create(<Provider store={store}>
      <BrowserRouter>
      <App />
      </BrowserRouter>
      </Provider>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
