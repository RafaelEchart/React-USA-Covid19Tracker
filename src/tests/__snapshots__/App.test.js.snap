// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders as intended App component matches the snapshot 1`] = `
<div
  className="App"
>
  <div
    className="contentContainer"
  >
    <div
      className="loadingCover"
    >
      <img
        src="american_flag.gif"
      />
      <div
        className="ant-spin ant-spin-lg ant-spin-spinning"
      >
        <span
          className="ant-spin-dot ant-spin-dot-spin"
        >
          <i
            className="ant-spin-dot-item"
          />
          <i
            className="ant-spin-dot-item"
          />
          <i
            className="ant-spin-dot-item"
          />
          <i
            className="ant-spin-dot-item"
          />
        </span>
      </div>
    </div>
  </div>
</div>
`;
